client/my-sites/plugins-wpcom/jetpack-plugin-item.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import PurchaseButton from './purchase-button';
+
+export const JetpackPluginItem = ( {
+	plugin,
+	siteSlug,
+	isActive,
+	translate = identity
+} ) => {
+	const translatePlan = {
+		premium: translate( 'Premium' ),
+		business: translate( 'Business' ),
+	};
+	const planClasses = classNames(
+		'button',
+		'is-compact',
+		'is-borderless',
+		`plugins-wpcom__is-${ plugin.plan }-plugin`
+	);
+	const plan = [ 'premium', 'business' ].includes( plugin.plan )
+		? <span className={ planClasses }>
+				{ translatePlan[ plugin.plan ] }
+			</span>
```

Comment:
usually _spanning_ multiple lines with JSX we like wrapping them in `()`s

also the lingering `null` here jumps out at me; I wonder if we could get rid of that